test/tests_for_db/ee2_MongoUtil_test.py

```diff
@@ -58,17 +58,55 @@ def test_init_ok(self):
         mongo_util = self.getMongoUtil()
         self.assertTrue(set(class_attri) <= set(mongo_util.__dict__.keys()))
 
+    def test_save_and_modify_updates_timestamp(self):
+        job = get_example_job(status=Status.created.value)
+        updated_ts = job.updated
+        job.save()
```

Comment:
Its seems a bit odd that saving a job updates the timestamp even if there are no changes, is that what you want?

Reply:
Well, updated tracks the last time the job was saved or modified, so seems ok to me. What do you suggest?

Reply:
I mean updated to me means that the job was updated, e.g. changed in some way. It doesn't make sense to me for the updated timestamp to change if the job hasn't changed

Reply:
Alright, so if we make sure to only save jobs with changes, then we should be alright?

Reply:
I'm not sure if its possible to know if a record has a different value or not when saving unless we reload the record or do two saves?

Reply:
> Alright, so if we make sure to only save jobs with changes, then we should be alright?

I think so, but we should document the behavior clearly

Reply:
> I'm not sure if its possible to know if a record has a different value or not when saving unless we reload the record or do two saves?

You can do a query on the entire record, but that's probably overkill. It'd be nice if Mongo had a feature where a field would only be updated if at least one of a set of provided fields changed